ra-optimisation/search-tree-rao/src/main/java/com/farao_community/farao/search_tree_rao/commons/objective_function_evaluator/MarginEvaluatorWithPstLimitationUnoptimizedCnecs.java

```diff
@@ -19,7 +19,9 @@
 import java.util.Map;
 
 /**
- * It enables to evaluate the absolute or relative minimal margin as a cost
+ * It enables to evaluate the absolute minimal margin as a cost
+ * For cnecs parameterized as in series with a pst, margin is updated
+ * when there are not enough setpoints on the pst left to absorb the cnec's margin deficit
```

Comment:
rather "margin is considered infinite if the PST has enough setpoints left to absorb the cnec's margin deficit"